torchci/pages/hud/[repoOwner]/[repoName]/[branch]/[[...page]].tsx

```diff
@@ -436,7 +436,12 @@ export default function Hud() {
     refreshInterval: 5 * 60 * 1000, // refresh every 5 minutes
   });
 
-  const viableStrictLag = data === undefined || data.length === 0 ? undefined : `viable/strict lag: ${data[0]["strict_lag_sec"]}h`;
+  let viableStrictLag: string | undefined;
+
+  if (data !== undefined && data.length > 0) {
+    const timeLagInHours = Math.floor(data[0]["strict_lag_sec"] / 3600);
```

Comment:
How do you feel about making it a bit more granular by adding a decimal place?

```suggestion
    const timeLagInHours = (data[0]["strict_lag_sec"] / 3600).toFixed(1);
```